services/movie_session.py

```diff
@@ -0,0 +1,49 @@
+from db.models import MovieSession
+from typing import Optional
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+    return new_movie_session
+
+
+def get_movies_sessions(session_date: Optional[str] = None) -> QuerySet:
+    scheduled_movies = MovieSession.objects.all()
+    if session_date is not None:
+        scheduled_movies = MovieSession.objects.filter(
```

Comment:
You can reuse your query:
```suggestion
        scheduled_movies = scheduled_movies.filter(
```